mypy/dmypy/client.py

```diff
@@ -265,12 +265,14 @@ def do_run(args: argparse.Namespace) -> None:
         # Bad or missing status file or dead process; good to start.
         start_server(args, allow_sources=True)
     t0 = time.time()
-    response = request(args.status_file, 'run', version=__version__, args=args.flags)
+    response = request(args.status_file, 'run', version=__version__, args=args.flags,
+                       no_tty=not sys.stdout.isatty())
```

Comment:
> Why not calculate it in `request` unless explicitly provided. Maybe just send it always to server, for consistency?

Yes, I was actually also not happy that `no_tty` in `**args` would not be statically checked. I will try to tighten this.
> Also style nit: I'd avoid an argument with a negated boolean value. This could be renamed to `is_tty`, for example, and it would arguably be cleaner.

OK, makes sense (also for the `no_color` below).